azure-pipelines.yml

```diff
@@ -45,12 +45,12 @@ jobs:
   #- powershell: .\build\Install-WindowsSdkISO.ps1 19041
   #  displayName: Insider SDK
 
-  - powershell: .\build\build.ps1 -target=Build
+  - powershell: .\build\build.ps1 -Target Build
```

Comment:
```suggestion
  - powershell: .\build\build.ps1 --Target=Build
```

Reply:
This new version of cake asks for -- to pass parameters to the build script.

Reply:
Seems it's not needed!